vision/recognition.proto

```diff
@@ -0,0 +1,27 @@
+
+syntax = "proto3";
+
+package admobilize_vision;
+
+// Describes a recognition feature descriptor
+message FeatureDescriptor {
+
+  // A unique identifier for the descriptor according to UUID v4
+  string uuid = 1; 
+
+  // Descriptor format used by extraction algorithm
+  string data_format = 2;
+
+  // The actual recognition descriptor data
+  repeated float data = 3;
+
+  // Arbitrary tags for the descriptor set
+  repeated string tags = 4;
+}
+
+// Container to group multiple FeatureDescriptors
+// makes storage of multiple FeatureDescriptors more
+// convenient
```

Comment:
More convenient or possible under X restriction? I don't remember the outcome of the discussion we had.